src/storage/heap_file.c

```diff
@@ -733,6 +734,23 @@ static int heap_stats_bestspace_finalize (void);
 static int heap_get_spage_type (void);
 static bool heap_is_reusable_oid (const FILE_TYPE file_type);
 
+// *INDENT-OFF*
+static SCAN_CODE heap_attrinfo_transform_fixed_to_disk (THREAD_ENTRY * thread_p, HEAP_CACHE_ATTRINFO * attr_info, OR_BUF *buf,
+							int index, std::set<int> *incremented_attrids, char *bitmap_bound);
+// *INDENT-ON*
+static SCAN_CODE heap_attrinfo_transform_variable_to_disk (THREAD_ENTRY * thread_p, HEAP_CACHE_ATTRINFO * attr_info,
+							   OR_BUF * buf, char **ptr_varvals, int index, int offset_size,
+							   int header_size, int lob_create_flag);
+static SCAN_CODE heap_attrinfo_transform_header_to_disk (THREAD_ENTRY * thread_p, HEAP_CACHE_ATTRINFO * attr_info,
+							 OR_BUF * buf, int offset_size, bool is_mvcc_class,
+							 bool has_prev);
```

Comment:
bool is_mvcc_class와 동일 형태이며, 직관적인 다음 명칭으로 변경 제안
```suggestion
							 bool is_update);
```

Reply:
반영했습니다.